js/ui/tool/dream.js

```diff
@@ -141,41 +141,7 @@ const _dream = async (endpoint, request) => {
 	let data = null;
 	try {
 		generating(true);
-		if (
```

Comment:
This was moved to clean up _dream() a little bit. As we only use txt2img in one case (dream tool when nothing is under the cursor), moving it there seems better.

Reply:
totally makes sense, i guarantee i was _scrambling_ to get something workable pushed out lol, i predicted HAVOC when i saw that A1111 commit ;)